### PR TITLE
feat: Add SelectionBar component :shell:

### PR DESCRIPTION
**Pull request by kosssi**
This is the SelectionBar on Cozy Drive:

capture d ecran 2017-06-08 a 15 17 35

This is the SelectionBar on Cozy Drive mobile:

capture d ecran 2017-06-08 a 15 18 07

With `cozy-ui/react/SelectionBar` you can:

```javascript
// import component
import { SelectionBar } from 'cozy-ui/react/SelectionBar'

// use component
<SelectionBar t={t} selected={seleted} actions={actions} hideSelectionBar={hideSelectionBar} />

// define actions
const mapDispatchToProps = dispatch => ({
  actions: {
    download: {
      action: selections => dispatch(downloadSelection(selections))
    },
    rename: {
      action: selections => dispatch(startRenamingAsync(selections[0])),
      displayCondition: selections => selections.length === 1
    },
    yourOwnAction: {
      action: selections => dispatch(yourOwnAction(selections),
    },
    ...
  }
})
```

### CODE REVIEW

**Comment by ptbrowne** [comment]
What does it look like ?

File: react/selection/index.jsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+export const hideSelectionBar = () => ({ type: HIDE_SELECTION_BAR })
+export const toggleSelectionBar = () => ({ type: TOGGLE_SELECTION_BAR })
+export const toggleItemSelection = (item, selected) => ({ type: selected ? UNSELECT_ITEM : SELECT_ITEM, id: item.id })
+
+// components
+export { SelectionBar }
+
+// reducers
+const selected = (state = [], action) => {
+  if (action.meta && action.meta.cancelSelection) {
```
Comment (ptbrowne) [inline react/selection/index.jsx]: More simply `action.cancelSelection` ?

**Comment by ptbrowne** [comment]
I agree with @goldoraf, a user of the SelectionBar should not have to worry about redux.

**Comment by kosssi** [comment]
I update this PR to add only an dump component.

**Pull request merged**